lib/gui/utils/byte-size/byte-size.js

```diff
@@ -29,7 +29,7 @@ const ByteSize = angular.module(MODULE_NAME, []);
 
 /* eslint-disable lodash/prefer-lodash-method */
 
-ByteSize.filter('gigabyte', require('./filters/gigabyte'));
+ByteSize.filter('closestUnit', require('./filters/closest-unit'));
```

Comment:
As there'll (presumably) now only be ever one filter, could/should `lib/gui/utils/byte-size/filters/` be removed?